src/components/charts/sankey/sankey-node/sankey-node-component.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Rectangle, Layer } from 'recharts';
+import { PropTypes } from 'prop-types';
+import styles from './sankey-node-styles.scss';
+
+function SankeyNode({ x, y, width, height, index, payload, config, containerWidth }) {
+  const padding = 20;
+  const isOut = x + width + padding > containerWidth;
+  const unit = config.unit ? `${config.unit} ` : '';
+  const suffix = config.suffix ? ` ${config.suffix}` : '';
+  return (
+    <Layer key={`CustomNode${index}`}>
+      <Rectangle
+        x={x}
+        y={y}
+        width={width}
+        height={height}
+        fill={payload.color}
+        fillOpacity="1"
+      />
+      <text
+        textAnchor={isOut ? 'end' : 'start'}
+        x={isOut ? x - padding : x + width + padding}
+        y={y + height / 2}
+        className={styles.nodeText}
+      >
+        {`${payload.name}: ${unit}${payload.value}${suffix}`}
+      </text>
+    </Layer>
+  );
+}
+
+SankeyNode.propTypes = {
+  x: PropTypes.number,
+  y: PropTypes.number,
+  width: PropTypes.number,
+  height: PropTypes.number,
+  index: PropTypes.number,
+  payload: PropTypes.object,
+  config: PropTypes.object,
```

Comment:
Would be nice to have `config` as a `PropTypes.shape` to be able to see the internal structure of the object (`unit` and `suffix`) as in `sankey-tooltip-component`.
Is it possible to display nested shapes on the github pages? Would be awesome to have a grasp of the structure of the objects just checking it on the styleguidist.

Reply:
It is, like so:
![image](https://user-images.githubusercontent.com/6136899/45287468-43533280-b4e0-11e8-8db5-550d83801d9b.png)
under `Props & Methods`